components/Escribinos/ContactForm.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+
+import { postClient } from "lib/sanity-client";
+
+interface ContactFormProps {
+  children: React.ReactNode;
+}
+
+export const ContactForm = ({ children }: ContactFormProps) => {
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    const form = event.currentTarget as HTMLInputElement;
+    const response = await postClient.create({
+      _type: "message",
+      // @ts-expect-error TODO: Correct type
+      name: form.name.value,
+      // @ts-expect-error TODO: Correct type
+      email: form.email.value,
+      // @ts-expect-error TODO: Correct type
+      body: form.body.value,
```

Comment:
**Note:** I didn't find the correct types, does anyone have suggestions?

Reply:
@LPRegen el type para el `event` es `React.FormEvent<HTMLFormElement>`, con eso ya no necesitarás hacer type casting acá:
```ts
const form = event.currentTarget as HTMLInputElement;
```

Y si te animas, puedes agregar la funcionalidad para que se muestre componente para el Callout (en el mismo archivo del form) una vez que se envíe el formulario, o también, se puede dejar eso como un issue 

---

Btw, recién recuerdo que lo que te quería decir sobre el `FormData`, la cosa era así:
```ts
const formData = new FormData(event.currentTarget);
const data = Object.fromEntries(form); // Esta data idealmente tendriamos que validarla, luego.
```